posthog/models/experiment.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
```

Comment:
@neilkakkar @EDsCODE thoughts on attributes to add or remove from the model?

Reply:
While we're not going to touch this right now, an experiment can have metadata like: confidence interval, significance score calculation method, etc. etc.

I think adding a `hyperparameters` dict makes sense here? You wouldn't ever query using this, so it being a JSON is fine, don't need an index on it.

Reply:
`experiment start date` being one of the pertinent params here^

Reply:
Actually, `experiment_start_date` should be outside, a standalone column.

Reply:
For historical experiments, `end_date` also makes sense actually. That way we don't update the insight to current date ranges, and always show it for the experiment date range.

(if you want to see what's happening today, create a new insight)